remove: step.Payload removed, since it was deprecated for a long time (#115)
ovh/utask: pkg/plugins/builtin/ssh/README.md

## Requirements

> None by default. SSH credentials should be retrieved from `{{.config.mySSHKey}}` rather than hardcoded on the template.

Reviewer: ```suggestion
None. SSH credentials should be retrieved from e.g. `{{.config.mySSHKey.privateKey}}` rather than hardcoded in the template.

Example configuration object __mySSHKey__ (yaml):
``
privateKey: |-
  -----BEGIN OPENSSH PRIVATE KEY-----
  ...
  -----END OPENSSH PRIVATE KEY-----
```

## Note

> The plugin returns two objects, the `output` of the execution regarding the selected `output_mode` in the configuration: either using the `result` configuration, or any given JSON in stdout/stderr inside the given starting/ending delimiters provided in configuration `output_manual_delimiters`.

Reviewer: ```suggestion
The plugin returns two objects, `output` and `metadata`.

`output` depends on the `output_mode` configuration. It is read as formatted JSON from the stdout of the SSH session. 
The default `output_mode` (`auto-result`) injects an escaped `echo` command as the last command in the SSH session, to craft a JSON payload where all the values are interpolated by the shell (e.g. shell variables get replaced). The format of this payload is configured via the `result` configuration. Other modes expect the JSON output on the last line (`output_last_line`) or between given delimiters (`output_manual_delimiters`).
```

}
```

> The `metadata` to fetch informations about plugin execution:

Reviewer: ```suggestion
`metadata` contains information about plugin execution:
```
